Library/Formula/sphinx.rb

```diff
@@ -4,8 +4,8 @@ class Sphinx < Formula
   revision 1
```

Comment:
Sorry, I didn't notice this before: this line needs remove as it's a new version. My apologises for the back-and-forth.